src/plugins/jma/datasource.py

```
@@ -0,0 +1,364 @@
+        
+    async def health_check(self) -> Dict[str, Any]:
+        try:
+            session = await self._get_session()
+            async with session.get(self.amedas_api_url) as response:
+                amedas_ok = response.status == 200
+                
+            return {
+                'status': 'healthy' if amedas_ok else 'degraded',
+                'amedas_api': 'available' if amedas_ok else 'unavailable',
+                'jra55': 'requires_manual_download',
+                'timestamp': datetime.now().isoformat()
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)